### PR TITLE
Chronos: Fixed the need to manually divide batch_size in `distributed`

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by TheaperDeng** [comment]
1. should divided by workers_per_node * OrcaContext.node_num
2. add document to illustrate what will be done to the input batch_size

File: python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py [hunk 1]
```diff
@@ -82,6 +82,10 @@ def fit(self, data, epochs=1, batch_size=32):
 
         # fit on internal
         if self.distributed:
+            if batch_size % self.workers_per_node != 0:
+                raise RuntimeError("Please make sure that 'batch_size' "
```
Comment (jason-dai) [inline python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py:86]: In the error message, print the value of `batch_size` and `worker_per_node`.

Comment (liangs6212) [reply python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py:86]: Okay, I will add relevant values ​​in the next submission

File: python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py [hunk 2]
```diff
@@ -82,6 +83,17 @@ def fit(self, data, epochs=1, batch_size=32):
 
         # fit on internal
         if self.distributed:
+            # for cluster mode
+            from bigdl.dllib.nncontext import init_nncontext
+            nn = init_nncontext().getConf()
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py:88]: sc = OrcaContext.get_spark_context()

File: python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py [hunk 3]
```diff
@@ -66,7 +66,8 @@ def fit(self, data, epochs=1, batch_size=32):
                | should follow the shape stated before.
 
         :param epochs: Number of epochs you want to train. The value defaults to 1.
-        :param batch_size: Number of batch size you want to train. The value defaults to 32.
+        :param batch_size: Number of batch size you want to train. The value defaults to 32,
+               if 'distributed', the data divided by batch_size is evenly distributed to all node.
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py:70]: if the forecaster is distributed, the batch_size will be evenly distributed to all workers.

**Review by TheaperDeng — APPROVED** [review]
LGTM, jenkins

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/34/

**Pull request merged**